src/middlewares/authorization.js

```diff
@@ -0,0 +1,16 @@
+import verifyToken from '../utils/Authentication';
+
+const checkToken = (req, res, next) => {
+  const token = req.headers['x-access-token'] || req.query.token || req.headers.authorization;
+  if (token) {
+    const response = verifyToken(token);
+    if (response.success === true) return next();
```

Comment:
`if (response.success === true)` is better written as `if (response.success)`.
Also, your code did not cater for when success is false. why is this so?

Reply:
If it's false, the response object already has an error property with a message in the Authentication.js file